### PR TITLE
Update what-is-azure-pipelines.md

### PR DESCRIPTION
**Pull request by bryan-strassner**
Change references to TFS to Azure DevOps (ADO) Server to match product rename

### CODE REVIEW

**Comment by PRMerger17** [comment]
@bryan-strassner : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by juliakm** [comment]
#sign-off

File: docs/pipelines/get-started/what-is-azure-pipelines.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Azure Pipelines automatically builds and tests code projects to make them availa
 It works with just about any language or project type. Azure Pipelines combines continuous integration (CI) and continuous 
 delivery (CD) to constantly and consistently test and build your code and ship it to any target. 
 
-Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in TFS helps you set up and manage CI for your applications.
+Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in Azure DevOps (ADO) Server helps you set up and manage CI for your applications.
```
Comment (ShannonLeavitt) [inline docs/pipelines/get-started/what-is-azure-pipelines.md:18]:
```suggestion
Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in Azure DevOps Server helps you set up and manage CI for your applications.
```
Removing acronym, per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=44177)

File: docs/pipelines/get-started/what-is-azure-pipelines.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ Azure Pipelines automatically builds and tests code projects to make them availa
...
 
-Continuous Delivery (CD) is a process by which code is built, tested, and deployed to one or more test and production environments.  Deploying and testing in multiple environments drives quality.  CI systems produce the deployable artifacts including infrastructure and apps.  Automated release processes consume these artifacts to release new versions and fixes to existing systems.  Monitoring and alerting systems run continually to drive visibility into the entire CD process. The Release service in TFS helps you set up and manage CD for your applications.
+Continuous Delivery (CD) is a process by which code is built, tested, and deployed to one or more test and production environments.  Deploying and testing in multiple environments drives quality.  CI systems produce the deployable artifacts including infrastructure and apps.  Automated release processes consume these artifacts to release new versions and fixes to existing systems.  Monitoring and alerting systems run continually to drive visibility into the entire CD process. The Release service in ADO Server helps you set up and manage CD for your applications.
```
Comment (ShannonLeavitt) [inline docs/pipelines/get-started/what-is-azure-pipelines.md:20]:
```suggestion
Continuous Delivery (CD) is a process by which code is built, tested, and deployed to one or more test and production environments.  Deploying and testing in multiple environments drives quality.  CI systems produce the deployable artifacts including infrastructure and apps.  Automated release processes consume these artifacts to release new versions and fixes to existing systems.  Monitoring and alerting systems run continually to drive visibility into the entire CD process. The Release service in Azure DevOps Server helps you set up and manage CD for your applications.
```

File: docs/pipelines/get-started/what-is-azure-pipelines.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ Once you have continuous integration in place, the next step is to create a rele
 
 Whether your app is on-premises or in the cloud, you can automate build-deploy-test workflows and choose the technologies and frameworks, then [test your changes continuously](../ecosystems/dotnet-core.md#run-your-tests) in a fast, scalable, and efficient manner. 
 
-* Maintain quality and find problems as you develop. Continuous testing with TFS ensures your app still works after every check-in and build, enabling you to find problems earlier by running tests automatically with each build.
+* Maintain quality and find problems as you develop. Continuous testing with ADO Server ensures your app still works after every check-in and build, enabling you to find problems earlier by running tests automatically with each build.
```
Comment (ShannonLeavitt) [inline docs/pipelines/get-started/what-is-azure-pipelines.md:51]:
```suggestion
* Maintain quality and find problems as you develop. Continuous testing with Azure DevOps Server ensures your app still works after every check-in and build, enabling you to find problems earlier by running tests automatically with each build.
```

**Comment by PRMerger10** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**